app/models/group.rb

```diff
@@ -1,5 +1,9 @@
 class Group < ApplicationRecord
   belongs_to :user
+  store :days_of_the_week, accessors: [:sunday?, :monday?, :tuesday?, :wednesday?, :thursday?, :friday?, :saturday?]
 
   validates :from_time, :to_time, presence: true
+
+  has_many :group_members, dependent: :destroy
+  has_many :users, through: :group_members
```

Comment:
I tried to create a "members" relation but doesn't work for me, according to the documentation I tried with options like `:source, :as, and :class_name` but any of this doesn't work for me, so I decide leave like this and continues working.
examples that I tried:
`has_many :members, through: :group_members, class_name: "User"`
`has_many :members, through: :group_members, source: "User"`
`has_many :users, through: :group_members, as: "members"`